packages/adapter-cloudflare-workers/index.js

```diff
@@ -34,11 +34,16 @@ export default function (options) {
 			utils.log.minor('Generating worker...');
 			utils.copy(`${files}/entry.js`, '.svelte-kit/cloudflare-workers/entry.js');
 
+			writeFileSync('${files}/nofig.js', `export const generateNonces = ${config.kit.cspNonce}`);
```

Comment:
what is `nofig`? does this file get included somehow? I don't see anything referring to it. Can we give it a clearer name?

Reply:
I honestly don't know. Probably a typo for 'config'? I think this line was supposed to be removed actually, as the variable is passed in through esBuild.